internal/server/cache.go

```diff
@@ -0,0 +1,112 @@
+package server
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3" // Import sqlite3 driver for database/sql library
+	"github.com/metno/go-mms/pkg/mms"
+)
+
+const defaultDBFilePath = "/tmp/eventsCache.db"
```

Comment:
Find something more cross-platform

Reply:
This actually works on MacOS also. At least I could not reproduce it... strange.